Server/sprawl_calculator/calculator/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+#Url for endpoint1 and endpoint2
+urlpatterns = [
+    path('endpoint1/',endpoint1, name='endpoint1'),
+    path('endpoint2/',endpoint2, name='endpoint2'),
```

Comment:
Same as above the [second endpoint](https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L369) is `v1/wup_index/{processing_id}`